product_stock_location/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_product_stock_location_manager,stock.quant manager,model_product_stock_location,stock.group_stock_manager,1,1,1,1
```

Comment:
You don't need this since you are giving the same rights to the <code>stock_user</code>